README.md

```diff
@@ -20,3 +20,20 @@ Zebra is distributed under the terms of both the MIT license
 and the Apache License (Version 2.0).
 
 See [LICENSE-APACHE](LICENSE-APACHE) and [LICENSE-MIT](LICENSE-MIT).
+
+## Metrics
+
+Notes on local metrics collection:
+
+```
+# create a storage volume for grafana (once)
+sudo docker volume create grafana-storage
+# create a storage volume for prometheus (once)
+sudo docker volume create prometheus-storage
+
+# run prometheus with the included config
+sudo docker run --network host -v prometheus-storage:/prometheus -v /path/to/zebra/prometheus.yaml:/etc/prometheus/prometheus.yml  prom/prometheus
+
+# run grafana
+sudo docker run -d --network host -v grafana-storage:/var/lib/grafana grafana/grafana
+```
```

Comment:
I would add a line about 'open Grafana in a browser at localhost:3030' and maybe a link to the Prometheus metrics in Grafana page.

Reply:
Fixed!